README.md

```diff
@@ -518,11 +568,48 @@ Type: `config => transformedConfig`. Default `x => x` (identify function).
 If you want to make changes to the Webpack configuration beyond what's available in the above options, you can use this, the nuclear option.
 Your function receives the Webpack configuration that Underreact generates and returns a new Webpack configuration, representing your heart's desires.
 
-### vendorModules
+**Tip:** You should think twice before using `webpackConfigTransform`, as Underreact tries its best to abstract away Webpack so that you can focus on the important part of building your application.
 
-Type: `Array<string>`. Default: `[]`.
+## FAQs
 
-Identifiers of npm modules that you want to be added to the vendor bundle.
-The purpose of the vendor bundle is to deliberately group dependencies that change relatively infrequently — so this bundle will stay cached for longer than the others.
+### Why can't I import CSS?
 
-By default, the vendor bundle includes `react` and `react-dom`.
+### How do I deploy my code to a branch like `hey-pages`?
+
+### How do I access siteBasePath in my code?
+
+### Why not use `NODE_ENV`?
```

Comment:
I'm not sure this qualifies is a frequently asked question, as least for the users we're targeting, right?